asan/lsan.supp

```diff
@@ -4,77 +4,9 @@
 # File format:
 #leak:*
 
-# test: app/crypto.test.lua
-# source: /usr/lib/x86_64-linux-gnu/libcrypto.so
-leak:CRYPTO_zalloc
+leak:sql_prepare_and_execute
+leak:sqlParser
```

Comment:
Are there any plans to fix these leaks as well?

Reply:
Added an issue #10532:

```diff
diff --git a/asan/lsan.supp b/asan/lsan.supp
index 17e690abf3..e18e6bbe60 100644
--- a/asan/lsan.supp
+++ b/asan/lsan.supp
@@ -4,9 +4,10 @@
 # File format:
 #leak:*

+# to be removed in scope of gh-10532
 leak:sql_prepare_and_execute
 leak:sqlParser

-# FIXME suppress until gh-10184 is fixed
+# to be removed in scope of gh-10184
 leak:lua_yaml_encode
 leak:l_load
```